### PR TITLE
Add new commands and update test cases

### PR DESCRIPTION
**Pull request by pacoyang**

### CODE REVIEW

File: test/TestLensApiConsumerContract.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    const [deployer] = await ethers.getSigners();
+    const TestLensApiConsumerContract = await ethers.getContractFactory("TestLensApiConsumerContract");
+    const consumer = await TestLensApiConsumerContract.deploy(deployer.address);
+
+    // Make a request
+    const profileId = "0x01";
+    await expect(consumer.request(profileId)).to.emit(consumer, "MessageQueued");
+
+    // Start a localhost Phat functions server to execute the task
+    execSync(`yarn localhost-watch ${consumer.address} artifacts/contracts/TestLensApiConsumerContract.sol/TestLensApiConsumerContract.json dist/index.js -a https://api-mumbai.lens.dev/ --once`, { stdio: 'inherit' });
```
Comment (h4x3rotab) [inline test/TestLensApiConsumerContract.ts]: Is it possible to test with the local network + functions simulator by default?

File: src/index.ts [hunk 2]
```diff
@@ -111,7 +111,7 @@ function parseProfileId(hexx: string): string {
   hex = hex.slice(2);
   var str = "";
   for (var i = 0; i < hex.length; i += 2) {
-    const ch = String.fromCharCode(parseInt(hex.substring(i, 2), 16));
+    const ch = String.fromCharCode(parseInt(hex.substr(i, 2), 16));
```
Comment (h4x3rotab) [inline src/index.ts]: Isn't it a deprecated function? Btw, let's hide the "helper functions" in somewhere else, not inside the index.ts file to make it clearer.

File: .env.local [hunk 3]
```diff
@@ -23,3 +23,14 @@ MUMBAI_LENSAPI_ORACLE_ENDPOINT=0xa31cb575847033341a1af63ac86fee33818a4838
...
+# Don't touch it if you don't know what it is.
+PHALA_MAINNET_ENDPOINT=wss://api.phala.network/ws
+PHALA_TESTNET_ENDPOINT=wss://poc5.phala.network/ws
+
+# Don't touch it if you don't know what it is.
+PHAT_BRICKS_MAINNET_FACTORY_CONTRACT_ID=0xb59bcc4ea352f3d878874d8f496fb093bdf362fa59d6e577c075f41cd7c84924
+PHAT_BRICKS_TESTNET_FACTORY_CONTRACT_ID=0x489bb4fa807bbe0f877ed46be8646867a8d16ec58add141977c4bd19b0237091
+
+# The encypt passphrase for the exported polkadot wallet JSON file.
+POLKADOT_WALLET_PASSPHRASE=
```
Comment (h4x3rotab) [inline .env.local:36]: Suggest to allow loading mnemonic (sURI) by default. Json keystore should be the backup option.

File: src/index.ts (hunk 2 above)
Comment (Leechael) [reply src/index.ts]: `hex.substring(i, 3)`

File: .env.local (hunk 3 above)
Comment (Leechael) [reply .env.local:36]: Added.

**Pull request closed without merging**